### ISSUE TITLE
Why the icons on the buttons are so small ?

### ISSUE THREAD

**Issue by seba123neo** [mikepenz/Android-Iconics#46]
Hi, Mike, I'm using this library and everything is perfect, except for a small observation on the buttons.

I explain here:

I have in my application buttons 50dpi (width) and 50dpi(height), this size is perfect for showing the icons of google material design (PNG), (I put in all the folders, the icon with all resolutions drawable-MDPI, drawable- hdpi..etc ...)

the icon is displayed perfect and almost fills the size of the button.

but if I want to do the same with the button of this library, the icon is very small, almost half of the other.

to summarize: 
with a button of 50x50 and using PNG images the icon fills the button perfect.
but with the button of this library the icon can barely be seen (to small).

Is it possible to assign in the xml, the resolution of the icon to display on the button?
the icon is automatically adapted to the size of the button?

**Comment by mikepenz** [comment]
@seba123neo please add a screenshot

**Comment by seba123neo** [comment]
@mikepenz ok, one minute

**Comment by seba123neo** [comment]
here is the screenshot of the normal android button (AppCompatButton or Button is the same...)

all 50dpi x 50dpi (with PNG images)

![normal_button](https://cloud.githubusercontent.com/assets/9873666/8556288/80d0a7ca-24cc-11e5-8774-cf65188f49b4.png)

here is the same 50dpi x 50dpi, (cloud icon) with the iconicsButton, looks smaller...

![screenshot_2015-07-07-17-20-03](https://cloud.githubusercontent.com/assets/9873666/8556326/c63d90fc-24cc-11e5-9995-7bd83ff8a9b5.png)

**Comment by mikepenz** [comment]
how do you define the icon for the IconicsButton?

**Comment by seba123neo** [comment]
<com.mikepenz.iconics.view.IconicsButton
            android:id="@+id/btnEliminar"
            android:layout_width="50dp"
            android:layout_height="50dp"
            android:text="{gmd-cloud}"
            android:textAllCaps="false" />

**Comment by seba123neo** [comment]
another thing:

to add some color to the background of the buttons I'm using a simple drawable (first screenshoot), but if I want to use the drawable in the IconicsButton, the icon disappears.

**Comment by seba123neo** [comment]
the size of the icons in the first screenshoot is 24dp (is like the icons of the toolbar i think).

but in the second screenshoot the icons looks like 16dp

**Comment by mikepenz** [comment]
@seba123neo i will look at it. thanks for the details

**Comment by mikepenz** [comment]
@seba123neo if you use the IconicsButton as shown above you will have to set the TextSize for it. Because the icon is set as text. If you really want to have a different behavior you should go with an ImageButton and set the IconicsDrawable via code

**Comment by seba123neo** [comment]
thanks i set the text size in 24sp and now looks good...thanks !!!

**Comment by mikepenz** [comment]
@seba123neo make sure to set it as dp as "sp" will scale depending on the users font size setting on the device

**Comment by seba123neo** [comment]
Thanks, I will...

I saw that you have solved the issue of textAllCaps property, now no need to put on the button textAllCaps property right ?

**Comment by mikepenz** [comment]
@seba123neo correct
